### PR TITLE
Creating the README

### PR DESCRIPTION
**Pull request by isabellberzsenyi**
## Story Description

Create and update the README. The README contains information on:
- Starting instruction
- File structure
- Technologies
- Developer contact information

## Notes to Reviewer
File structure and technologies sections will be updated as the code repository works.

### CODE REVIEW

**Review by jzhzhang66 — APPROVED** [review]
This looks good to me!

File: README.md [hunk 1]
```diff
@@ -1,24 +1,76 @@
...
+
+```bash
+# In the project directory
+npm install
+```
+
+5. **Run the server locally**
 
 ```bash
 npm run dev
```
Comment (josephmannis) [inline README.md:50]: Would recommend yarn instead of npm because it can be more reliable for various things, but up to you.

**Review by josephmannis — CHANGES_REQUESTED** [review]
One other thing to add here would be the file structure and how your code is organized. Also, could you add some more specific instructions on how to deploy the site? 

Other than that looks good to me!

File: README.md [hunk 2]
```diff
@@ -1,24 +1,76 @@
...
+   In the project directory there is a file called `env_example`, which holds all the necessary environment variables. Most importantly, it contains the Prismic access token.
+
+Copy the file and add the values to the variables.
+
+```sh
+cp env_example .env
+```
+
+```sh
+PRISMIC_URL="XXXXXXXXXXXXXXX"
```
Comment (josephmannis) [inline README.md:37]: Also, can you change the paragraph to mention Prismic URL instead of Prismic access token?

**Review by jzhzhang66 — COMMENTED** [review]
This looks good - probably a typo but my email is jzhzhang66@gmail.com 
Otherwise, it looks like everything has been changed accordingly to the feedback from Joe.

**Review by josephmannis — APPROVED** [review]
🚀

**Pull request merged**